pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/extensions/ExtensibleLoadManagerImpl.java

```diff
@@ -380,12 +380,22 @@ public CompletableFuture<Optional<BrokerLookupData>> assign(Optional<ServiceUnit
     }
 
     public CompletableFuture<Optional<String>> selectAsync(ServiceUnitId bundle) {
+        return selectAsync(bundle, Optional.empty());
+    }
+
+    public CompletableFuture<Optional<String>> selectAsync(ServiceUnitId bundle,
+                                                           Optional<Set<String>> excludeBrokerSet) {
```

Comment:
maybe we can just use empty set like `Collections.emptySet()` to replace `Optional`

Reply:
+1

Reply:
Updated.